pymc3/variational/advi.py

```diff
@@ -303,7 +306,10 @@ def sample_vp(
     global_RVs = list(set(model.free_RVs) - set(rvs(local_RVs)))
 
     # Make dict for replacements of random variables
-    r = MRG_RandomStreams(seed=random_seed)
+    if random_seed is None:
+        r = MRG_RandomStreams()
```

Comment:
The default seed of MRG_RandomState() is a constant so its result will be the same regardless of the global random state of numpy. Do you intend such behavior if we do not set the seed exlicitly?